src/components/play/layer-settings/LayerSettings.js.old

```diff
@@ -0,0 +1,179 @@
+import React, { Component } from 'react'
```

Comment:
Why are you keeping this file?

Reply:
IDk why I initially kept it but I'm planning on deleting them all after the merge